plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -517,17 +517,17 @@ which is of function type but isn't a lambda until you reduce.
 which is a function type but isn't a function type until you reduce.
 
 It's usually okay to stop at this point, since the remaining things usually aren't "proper" arguments.
-In the term case, it's a lambda computed by an application, which won't occur form a "proper" argument.
-In the type case, we only generate type lambdas for newtypes, which will blcok "proper" arguments anyway,
+In the term case, it's a lambda computed by an application, which won't occur from a "proper" argument.
+In the type case, we only generate type lambdas for newtypes, which will block "proper" arguments anyway,
 i.e. it comes from something like this:
 
 f :: Identity (a -> a)
 f = Identity (\x -> x)
 -}
 
 -- | Add entry/exit tracing inside a term's leading arguments, both term and type arguments.
--- @(\a -> /\b -> body)@ into @\a -> /\b -> entryExitTracing body@.
--- @(\a -> /\b -> body)@ into @\a -> /\b -> entryExitTracing body@.
+-- @(\a -> \b -> body)@ into @\a -> \b -> entryExitTracing body@.
+-- @(/\a -> /\b -> body)@ into @/\a -> /\b -> entryExitTracing body@.
```

Comment:
Sorry maybe you just wanted to delete L530:
```suggestion
-- @(\a -> /\b -> body)@ into @\a -> /\b -> entryExitTracing body@.
```

Reply:
Oh yeah, I just had a duplicate line.